safe-global/web-core
# Refactor: simplify SignOrExecute props

## What it solves

`isExecutable={safe.threshold === 1}` was passed to SignOrExecuteForm in every modal except for the ConfirmTxModal. So I moved it into the SignOrExecuteForm itself if txId isn't passed (meaning it's a new tx).

Also I removed the redirect to tx on submit, because we now have a notification with the transaction link popping up after tx creation.

Also fixed that txId wasn't passed to the tx events in the on-chain signing flow.

## How to test

- Create a transaction in a 1/1 Safe, it should be immediately executable
- Create a tx in a 2/2 Safe, it should not be executable
- Confirm an existing signed transaction, it should be executable
- Confirm a signed tx with an lower/higher nonce than Safe nonce – it should not be executable
